src/openforms/forms/admin/__init__.py

```diff
@@ -2,6 +2,7 @@
 from . import form  # noqa
 from . import form_definition  # noqa
 from . import form_logic  # noqa
+from . import form_statistics  # noqa
```

Comment:
Note for later: I believe having an explicit `__all__` instead of the noqas would be better